test/Conversion/TorchToStd/basic.mlir

```diff
@@ -74,3 +74,14 @@ func @torch.constant.int() -> !torch.int {
   %int1 = torch.constant.int 1
   return %int1 : !torch.int
 }
+
+// CHECK-LABEL:  func @torch.aten.add.int(%arg0: !torch.int, %arg1: !torch.int) -> !torch.int {
+// CHECK:    %[[LHS_I64:.*]] = torch_c.to_i64 %[[LHS]]
```

Comment:
Are the CHECK tags generated from `generate-test-checks.py`? The space between CHECK tag and the content to check against are not the same as the previous tests.
```
// CHECK:           return %[[FLOAT]] : !torch.float
vs
// CHECK:    return %[[INT:.*]] : !torch.int
```

Reply:
Busted. I wrote them manually, thanks.